packages/icons/tests/chromatic/Icons.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import type { ContextValue } from "react-aria-components";
+import type { ResponsiveProp, ColorValue } from "@hopper-ui/styled-system";
+import * as IconLibrary from "@hopper-ui/icons";
+
+import "./icons.stories.css";
+
+interface IconLibraryType {
+    [key: string]: React.ComponentType<ContextValue<Omit<IconLibrary.IconProps, "src16" | "src24" | "src32" >, SVGSVGElement>>;
+}
+
+interface ListProps {
+    icons: string[];
+    size?: ResponsiveProp<"sm" | "md" | "lg">;
+    color?: ResponsiveProp<ColorValue>;
+}
+
+const List = ({ icons, size, color }: ListProps) => {
+    const listItems = icons.map(name => {
+        const Component = (IconLibrary as unknown as IconLibraryType)[name];
+
+        return (
```

Comment:
i would use the styled-system instead of creating an additionnal css file. 

```
<Li key={name}  display="flex"  flex-wrap="wrap" alignItems="center" gap="inset-md" margin="none" padding="none" style={{listStyle: "none" }}>
                <Component size={size} color={color} />
</Li>

return <Ul display="block">{listItems}</Ul>;
```

Reply:
really? I just figured it would look messy and add a bunch of html, but sure I can do this.